content/sensu-go/5.14/getting-started/learn-sensu.md

```diff
@@ -407,22 +396,31 @@ Then create the check using sensuctl and the `disk_usage-check.json` file includ
 sensuctl create --file disk_usage-check.json
 {{< /highlight >}}
 
-We should see it working in the [dashboard entity view](http://localhost:3002/#/entities) and via sensuctl:
+_**NOTE**: You don't need to make any changes to disk_usage-check.json before running `sensuctl create --file disk_usage-check.json`._
```

Comment:
We need to rename the `disk_usage-check.json` in the sandbox lesson if we want to make this line in a note. The current `_` after the word `disk` is causing the note to not be rendered correctly in browser.

Reply:
Thank you! I'll fix that (remove the note formatting)